packages/fxa-settings/src/components/ConnectedServices/index.tsx

```diff
@@ -128,6 +304,163 @@ export const ConnectedServices = () => {
+                <li>
+                  <label>
+                    <input
+                      type="radio"
+                      className="ltr:mr-2 rtl:ml-2 -mt-1 align-middle"
+                      value="lost"
+                      name="reason"
+                    />
+                    Lost or Stolen
+                  </label>
+                </li>
+                <li>
+                  <label>
+                    <input
+                      type="radio"
+                      className="ltr:mr-2 rtl:ml-2 -mt-1 align-middle"
+                      value="old"
+                      name="reason"
+                    />
+                    Old or Replaced
+                  </label>
+                </li>
+                <li>
+                  <label>
+                    <input
+                      type="radio"
+                      className="ltr:mr-2 rtl:ml-2 -mt-1 align-middle"
+                      value="duplicate"
+                      name="reason"
+                    />
+                    Duplicate
+                  </label>
+                </li>
+                <li>
+                  <label>
+                    <input
+                      type="radio"
+                      className="ltr:mr-2 rtl:ml-2 -mt-1 align-middle"
+                      value="no"
+                      name="reason"
+                    />
+                    Rather not say
+                  </label>
+                </li>
+              </ul>
+            </Modal>
+          </VerifiedSessionGuard>
+        )}
+
+        {adviceModalRevealed && (
+          <Modal
+            onDismiss={onCloseAdviceModal}
+            onConfirm={onCloseAdviceModal}
+            confirmBtnClassName="cta-primary"
+            hasCancelButton={false}
+            confirmText="Okay, got it"
+            headerId="connected-services-advice-modal-header"
+            descId="connected-services-advice-modal-description"
+          >
+            {DS.reason === 'lost' ? (
+              <>
+                <h2
+                  id="connected-services-advice-modal-header"
+                  className="font-bold text-xl text-center mb-2"
+                  data-testid="connected-services-lost-device-modal-header"
+                >
+                  Lost or stolen device disconnected
+                </h2>
+                <p
+                  id="connected-services-advice-modal-description"
+                  data-testid="lost-device-desc"
+                  className="my-4 text-center"
+                >
+                  Since your device was lost or stolen, to keep your information
+                  safe, you should change your Firefox account password in your
+                  account settings. You should also look for information from
+                  your device manufacturer about erasing your data remotely.
+                </p>
+              </>
+            ) : (
+              /* DS.reason === 'suspicious' */ <>
```

Comment:
Commented out for debugging purposes?

Reply:
That was just trying to leave a little information for future selves, but I can remove it